common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisExteriorManager.java

```diff
@@ -43,10 +43,10 @@ public void setLocked(boolean locked) {
             return;
         }
         this.locked = locked;
-        if (this.getLastKnownLocation() != null) {
-            TardisNavLocation lastKnownLocation = this.getLastKnownLocation();
-            Level level = lastKnownLocation.getLevel();
-            BlockPos extPos = lastKnownLocation.getPosition();
+        if (this.operator.getPilotingManager() != null) {
```

Comment:
Code should make pilotingManager a existing variable, null check that and then use it in code